packages/devextreme/testing/tests/DevExpress.ui.widgets/menu.tests.js

```diff
@@ -2224,6 +2224,32 @@ QUnit.module('keyboard navigation', {
         assert.notOk($items.eq(0).hasClass(DX_STATE_FOCUSED_CLASS), 'first item lose focus');
     });
 
+    checkStyleHelper.testInChromeOnDesktopActiveWindow('root item text should not use color property from template when focused (T1227670)', function(assert) {
+        const $rootMenuItem = $(this.instance.itemElements().eq(0));
+
+        this.instance.focus();
+
+        assert.ok($rootMenuItem.hasClass(DX_STATE_FOCUSED_CLASS), 'root item should have focused class');
+        assert.strictEqual(checkStyleHelper.getColor($rootMenuItem[0]), 'rgb(51, 51, 51)', 'color');
+    });
+
+    [
+        [{ text: 'item1', items: [{ name: 'item_1_1' }] }],
+        [{ icon: 'imageCssClass', items: [{ name: 'item_1_1' }]}],
+        [{ text: 'item1', icon: 'imageCssClass', items: [{ name: 'item_1_1' }]}],
+    ].forEach(item => {
+        checkStyleHelper.testInChromeOnDesktopActiveWindow('root item text should not use color property from template when focused (T1227670)', function(assert) {
```

Comment:
what template?

Reply:
How is it related to a styling?

Reply:
The bug that we're dealing right now, without our fix, the style that it uses is from the menu item template. As you can see on the first assert rgb(255, 255, 255), that's the same style that it uses the causes the disappearance.

<img width="957" alt="Screenshot 2024-04-15 at 10 02 27 PM" src="https://github.com/DevExpress/DevExtreme/assets/133762589/a00fc50b-9c18-4b34-b12b-5094b100aa8c">